tests/mount_test.c

```diff
@@ -120,14 +120,14 @@ Suite* make_mount_suite(void) {
 	return s;
 }
 
-gboolean enable_debug = true;
 
```

Comment:
Well, previously I think it went
`}`
`  blank`
`  gboolean`
`  blank`
`  main`

and now I think it goes::

`  }`
`  blank`
`  blank`
`  main`

so, we have two consecutive blank lines - which doesn't fit with the rest of the code style :-) if you see what I mean?